src/styles/card.scss

```
@@ -0,0 +1,89 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

add `padding: 32px 16px 16px;` to avoid using right/left margin for children
```suggestion
  border-radius: 5px;
  padding: 32px 16px 16px;
```

---

ok